cpp/src/arrow/array/array_nested.h

```diff
@@ -179,12 +187,15 @@ class ARROW_EXPORT LargeListArray : public BaseListArray<LargeListType> {
   /// \param[in] pool MemoryPool in case new offsets array needs to be
```

Comment:
CI is complaining that you didn't document the new parameters here...
https://github.com/apache/arrow/runs/8012669293?check_suite_focus=true#step:6:2925